src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
```

Comment:
to też, niepotrzebujesz już jakiegoś importa to go usuń, albo włącz automatyczne usuwanie. Kliknij na to i Alt+enter -> optimize imports on the fly